doc/radosgw/multisite.rst

```diff
@@ -528,6 +528,16 @@ running the following commands on the object gateway host:
    systemctl start ceph-radosgw@rgw.`hostname -s`
    systemctl enable ceph-radosgw@rgw.`hostname -s`
 
+If you have used the ``cephadm`` command, you will not be able to use
```

Comment:
If this cluster was deployed with ``cephadm``,

Reply:
Accepted with slight modification to reduce ambiguity.